src/frida/repl.py

```diff
@@ -423,6 +431,57 @@ def _create_repl_script(self):
                 with codecs.open(self._user_script, 'rb', 'utf-8') as f:
                     user_script = f.read().rstrip("\r\n") + "\n\n// Frida REPL script:\n"
 
+            if self._codeshare_uri is not None:
+                trust_store = self._get_or_create_truststore()
+                project_url = 'https://codeshare.frida.re/api/project/{}/'.format(self._codeshare_uri)
```

Comment:
Agreed! That's normally what I try to do actually, but I do occasionally mix them up so your point might be valid :-P. Either way I'll make sure to pay better attention for that in future commits!